synapse/storage/events.py

```diff
@@ -389,6 +399,25 @@ def _persist_events(self, events_and_contexts, backfilled=False,
                                 if all_single_prev_not_state:
                                     continue
 
+                            state_delta_counter.inc()
+                            if len(new_latest_event_ids) == 1:
+                                state_delta_single_event_counter.inc()
+
+                                # This is a fairly handwavey check to see if we could
+                                # have guessed what the delta would have been when
+                                # processing one of these events.
+                                # What we're interested in is if the latest extremities
+                                # were the same when we created the event as they are
+                                # now. We guess this by looking at the prev events and
+                                # checking if they match up, as when this server creates
```

Comment:
what do you mean by "this server" ?

Reply:
As in, we are creating a new event locally, as opposed to coming in from federation

Reply:
I could change it to say "when we create an event locally"? Not sure if that is less confusing or not.